repository/Seaside-Core.package/WARequest.class/instance/forwarded.st

```diff
@@ -0,0 +1,3 @@
+accessing-headers
+forwarded
+	^ self headerAt: 'forwarded' ifAbsent:[ self headerAt: 'x-forwarded-for' ]
```

Comment:
I don't think this will work for the `Forwarded` header, as the contents would be `for=192.0.2.43, for="[2001:db8:cafe::17]"` or `for=123.34.567.89`. So we need to extract the first occurrence of the `for=` attribute.

Reply:
My intention is that this accessor returns the entire header (as a string). We don't try to parse it and create a `WARemoteAddress` object (or something like that). Since none of the uses in the protection filter or the ip-based tracking strategy really require to parse it as an address, but rather only use it as a value to index on or compare, I think that should be sufficient.